files/scripts/ohpc_install.sh

```diff
@@ -0,0 +1,199 @@
+	internal_netmask='255.255.0.0'
+	internal_broadcast='10.40.255.255'
+	internal_gateway='10.40.0.1'
+	internal_dns=${internal_gateway}
+	internal_domain_name=''
+	# Enable components optionis
+	enable_beegfs_client=False
+	enable_mpi_defaults=True
+	enable_mpi_opa=False
+	enable_clustershell=True
+	enable_ipmisol=False
+	enable_opensm=False
+	enable_ipoib=False
+	enable_ganglia=False
+	enable_genders=False
+	enable_kargs=False
+	enable_lustre_client=False
+	enable_mrsh=False
+	enable_nagios=False
+	enable_powerman=False
+	enable_intel_packages=False
+	enable_dhcpd_server=False
+	enable_ifup=False
+	enable_warewulf=False
+	enable_nfs_ohpc=True
+	enable_nfs_home=True
+	enable_helloworld=True
+
+else
+	echo 'Not implemented yet !!'
+fi
+
+if [ -d ${WORKSPACE}/ansible-playbook-for-ohpc ]; then
+    rm -rf ${WORKSPACE}/ansible-playbook-for-ohpc
+fi
+git clone -b ${git_branch} https://github.com/Linaro/ansible-playbook-for-ohpc.git ${WORKSPACE}/ansible-playbook-for-ohpc
+
+
+# Retrieve the slurm.conf file
+if [ ${slurmconf} != '' ]; then
+	olddir=$(pwd)
+	cd ${WORKSPACE}/ansible-playbook-for-ohpc/roles/slurm-client/files
+	sftp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no 10.40.0.13:${slurmconf}
+	mv $(basename ${slurmconf}) slurm.conf
+	cd ${olddir}
+fi
+
+cat << EOF > ${WORKSPACE}/hosts
+[sms]
+${master_ip}
+[cnodes]
+${cnode01ip}
+${cnode02ip}
+${cnode03ip}
+[ionodes]
+${master_ip}
+[devnodes]
+${master_ip}
+EOF
+
+
+cat << EOF > ${WORKSPACE}/ohpc_installation.yml
+sms_name: ${master_name}
+sms_ip: ${master_ip}
+sms_eth_internal: ${master_eth_internal}
+internal_network: ${internal_network}
+internal_netmask: ${internal_netmask}
+internal_broadcast: ${internal_broadcast}
+internal_gateway: ${internal_gateway}
+internal_domain_name: ${internal_domain_name}
+internal_domain_name_servers: ${internal_dns}
+eth_provision: ${eth_provision}
+cnode_eth_internal: ${eth_provision}
+enable_beegfs_client: ${enable_beegfs_client}
+enable_mpi_defaults: ${enable_mpi_defaults} 
+enable_mpi_opa: ${enable_mpi_opa}
+enable_clustershell: ${enable_clustershell}
+enable_ipmisol: ${enable_ipmisol}
+enable_opensm: ${enable_opensm}
+enable_ipoib: ${enable_ipoib}
+enable_ganglia: ${enable_ganglia}
+enable_genders: ${enable_genders}
+enable_kargs: ${enable_kargs}
+enable_lustre_client: ${enable_lustre_client}
+enable_mrsh: ${enable_mrsh}
+enable_nagios: ${enable_nagios}
+enable_powerman: ${enable_powerman}
+enable_intel_packages: ${enable_intel_packages}
+enable_dhcpd_server: ${enable_dhcpd_server}
+enable_ifup: ${enable_ifup}
+enable_warewulf: ${enable_warewulf}
+enable_nfs_ohpc: ${enable_nfs_ohpc}
+enable_nfs_home: ${enable_nfs_home}
+enable_helloworld: ${enable_helloworld}
+kargs: ${kargs}
+additional_modules: ${additional_modules}
+num_computes: ${num_compute}
+compute_regex: ${compute_regex}
+compute_prefix: ${compute_prefix}
+compute_nodes:
+- { num: 1, c_name: "${cnode01}", c_ip: "${cnode01ip}", c_mac: "${cnode01mac}", c_bmc: "10.41.1.0"}
```

Comment:
10.41.1.0?

Reply:
Warewulf asks for BMC IPs, so I feed him a dummy one.